shell/monitor/monitor.sh

```diff
@@ -2,11 +2,16 @@
 
 declare _monitor_pid _monitor_experiment_id _monitor_log_dir _monitor_update_es _monitor_current_test_id
 
-declare -r _monitor_eshost=aloja.bsc.es:9200
+declare -r _monitor_eshost=aloja.bsc.es
```

Comment:
Looks like we always prepend `https://` to this, so could just include the protocol as part of the variable.